ontologies/Sierra_film.json

```diff
@@ -0,0 +1,180 @@
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "8rm28aw",
+  "type": "Work",
+  "identifiers": [
+    {
+      "label": "b24993207",
+      "authority": "Sierra"
+    }
+  ],
+  "label": "Are health tests a good idea? An Horizon special",
+  "creators": [
+    {
+      "label": "Davies, Serena, television producer, television director",
```

Comment:
At some point, do we want the Agent ontology to have some notion of "role"? So this could eventually become something like:

```json
{
  "label": "Davies, Serena",
  "role": [
    "television producer",
    "television directory"
  ],
  "type": "Agent"
}
```

So if Serena has contributed in another capacity, there's still a way to find all their images in one go.

(I’m aware CALM doesn’t actually expose that info.)